helm-charts/charts/kube-starrocks/values.yaml

```diff
@@ -19,6 +19,8 @@ nameOverride: ""
 initPassword:
   enabled: false
   password: ""
+# TimeZone is used to set the environment variable TZ for pod, with Asia/Shanghai as the default.
+timeZone: Asia/Shanghai
```

Comment:
set to UTC by default. :)

Reply:
refer to：https://docs.mirrorship.cn/zh-cn/latest/administration/timezone#%E9%BB%98%E8%AE%A4%E6%97%B6%E5%8C%BA
The default value of the time_zone parameter is Asia/Shanghai.
Should the configuration here be consistent?